### ISSUE TITLE
Segmentation tab does not work correctly when more than 16 series are selected.

### ISSUE THREAD

**Issue by brikeats** [fedorov/PCampReview#111]
Selecting more than 16 series causes a divide by zero error on line 1407. This is because the `if` statement on lines 1399-1406 does't catch the case when `nVolumeNodes > 16`.

**Comment by fedorov** [comment]
Good catch! I didn't test with such configuration. Can you propose a patch that will make it work with more than 16 series? It will be a very crowded view, so please choose something that you like - I don't think we ever considered looking at that many series at once, just because of the screen size limitation.

**Comment by brikeats** [comment]
Maybe it make more sense to restrict the number of selectable series? Or just have it default to 4x4 for more than 16 series? As you said, with more than 4x4, it gets too crowded to be useful.

**Comment by fedorov** [comment]
Both showing less viewers than series and restricting the maximum number of selected series will be a bit tricky to handle (not intuitive, additional notifications, logic). I think it is better to create enough viewers to fit all series, and if the views are too small, user can go back and unselect.

**Comment by fedorov** [comment]
I meant to say, to fit all selected series.

**Comment by fedorov** [comment]
@brikeats are you still working on this project?

**Comment by brikeats** [comment]
@fedorov sorry, no. I'm no longer at UChicago (joined a startup called Uptake), and I doubt I'll have the time.

**Comment by fedorov** [comment]
@brikeats thanks for the update! Is there another person in Aytek related group involved with it, or you don't know? Just trying to get an idea how much coordination we need to do with the Aytek's group, and who is the contact (if any).

**Comment by brikeats** [comment]
@fedorov They didn't have a replacement for me when I left (~6 weeks ago), and I haven't heard anything about it since. you're probably best off asking Aytek or Dr. Runesha directly about their plans for this project.

**Comment by fedorov** [comment]
@brikeats alright, thanks! Good luck at the new job! 

p.s. You should read this book, if you haven't yet: https://www.amazon.com/Lean-Startup-Entrepreneurs-Continuous-Innovation/dp/0307887898
